core/modules/czar/CzarConfig.cc

```diff
@@ -68,7 +68,9 @@ CzarConfig::CzarConfig(util::ConfigStore const& configStore)
     : _mySqlResultConfig(configStore.get("resultdb.user", "qsmaster"),
             configStore.getRequired("resultdb.passwd"),
             configStore.getRequired("resultdb.host"), configStore.getInt("resultdb.port"),
-            configStore.getRequired("resultdb.unix_socket"), configStore.get("resultdb.db","qservResult")),
+            configStore.getRequired("resultdb.unix_socket"),
+            configStore.get("resultdb.db","qservResult"),
+            configStore.getInt("resultdb.maxtablesize", 5001)),
```

Comment:
Is 5001 the default value?
A reason why I'm asking is that there is some other default value of 5000 in another file. Could it this be streamlined a bit via some common macro definition, or some utility class with defaults? Otherwise we'll see a propagation of hard-wired limits for the same parameter throughout the code.